src/renderers/dom/shared/__tests__/CSSPropertyOperations-test.js

```diff
@@ -108,75 +108,103 @@ describe('CSSPropertyOperations', function() {
   });
 
   it('should warn when using hyphenated style names', function() {
+    var Comp = React.createClass({
+      displayName: 'Comp',
+      render: function() {
+        return <div style={{ 'background-color': 'crimson' }}/>;
+      },
+    });
     spyOn(console, 'error');
-
-    expect(CSSPropertyOperations.createMarkupForStyles({
-      'background-color': 'crimson',
-    })).toBe('background-color:crimson;');
-
+    var root = document.createElement('div');
+    ReactDOM.render(<Comp />, root);
     expect(console.error.argsForCall.length).toBe(1);
+    expect(console.error.argsForCall[0][0]).toContain('Check the render method of `Comp`');
     expect(console.error.argsForCall[0][0]).toContain('backgroundColor');
```

Comment:
Let's get rid of the 2 separate checks to look at the same warning and just `expect().toBe(full message)`. Same in the few cases below. I know it's a bit churny but ultimately better. Thanks!